libsolidity/analysis/PostTypeChecker.cpp

```diff
@@ -31,106 +31,175 @@ using namespace dev;
 using namespace langutil;
 using namespace dev::solidity;
 
+// Defined at the bottom to avoid declaring all Checkers at the top
+vector<shared_ptr<PostTypeChecker::Checker>> initializeCheckers(ErrorReporter& _errorReporter);
```

Comment:
Put inside anonymous namespace?

Reply:
Another way would be to define the constructor further down...

Reply:
I didn't want to split the class member definitions into several blocks